js/main.js

```diff
@@ -62,35 +61,41 @@ function call_warmly(search_criteria) {
           chrome.runtime.sendMessage('papmjbnpmffiahcnakjfjoobkefaemii', payload);
       },
       'error' : function(jqXHR, status, message) {
-          console.log('====> service call error: ' + status + ': ' + message); 
+          console.log('====> service call error: ' + status + ': ' + message);
           data = {"results":[]};
-          data.search_criteria = new Object(); 
+          data.search_criteria = new Object();
           data.search_criteria.target = 'Unable to analyze target due to various issues. Often times it works if you try again.';
           data.search_criteria.tags = '';
           data.search_criteria.closing = '';
-          chrome.runtime.sendMessage('papmjbnpmffiahcnakjfjoobkefaemii', 
+          chrome.runtime.sendMessage('papmjbnpmffiahcnakjfjoobkefaemii',
              {type: 'warmly_display_result', data: data});
-      }         
+      }
   });
 }
 
 // RegEx to parse recipient email in form of 'First Last <someemail@domain.foo.co.uk>'
 // We just want the name portion....
 // If these two declarations are moved to the top of this file
-// it breaks.  
+// it breaks.
 var email_re=/([\w\s,\._-]*)<([-.\w]+@[\w-]+(\.+[\w-]+)*)>/
 var recipient_name=null;
 
+function isRegisteredEmail(email) {
+  // Add our registered email list here
+  var emails = ["ghutirea@gmail.com"];
+  return emails.includes(email);
```

Comment:
Here we hard code our emails to make it look like we have something that does registration. SO put your email here if you don't want to see the registration page.

Reply:
sorry about those trailing whitespace issues...